Trev8939/statistical query simple

TADraeseke opened the pull request:
Adds a sample demonstrating basic functionality of a statistical query

https://devtopia.esri.com/runtime/common-samples/issues/425

shellygill commented on java/statistical-query/src/main/java/com/esri/arcgisruntime/sample/statisticalquery/MainActivity.java:
```diff
@@ -0,0 +1,171 @@
+
+        // show the results in a snackbar
+        Snackbar reportSnackbar = Snackbar.make(findViewById(R.id.activityMain), result, Snackbar.LENGTH_INDEFINITE);
+        reportSnackbar.setAction("Dismiss", view -> reportSnackbar.dismiss());
```
Strictly speaking, material design guidelines say don't use a Snackbar with Dismiss or Cancel as the action text. Could change to 'New query' or 'Change query'?

shellygill commented on java/statistical-query/src/main/java/com/esri/arcgisruntime/sample/statisticalquery/MainActivity.java:
```diff
@@ -0,0 +1,171 @@
+
+  private void getStatistics() {
+
+    // create definitions for each statistic to calculate
```
This section of code runs every time the user hits the query button, but it's exactly the same each time (setting up statistic definitions). Any reason why the `List<StatisticDefinition>` is not created once and stored?

TADraeseke commented:
Thanks @shellygill! 

Moved the whole creation of the StatisticsQueryParameters to outside of the getStatistics(...) method.

Also changed name of Snackbar dismissal button to 'New Query'
